apps/frontend-manage/src/components/evaluation/elements/ChoicesEvaluation.tsx

```diff
@@ -18,29 +25,43 @@ function ChoicesEvaluation({
   chartType,
   showSolution,
 }: ChoicesEvaluationProps) {
+  const [isCollapsed, setIsCollapsed] = useLocalStorage(
+    `evaluation-${instanceEvaluation.id}-isCollapsed`,
+    false
+  )
+
   return (
-    <>
-      <div className="order-2 flex-1 px-4 md:order-1">
+    <ResizablePanelGroup direction="horizontal">
+      <ResizablePanel
+        defaultSize={isCollapsed ? 100 : 80}
+        minSize={50}
+        className="px-4"
+      >
         <ElementChart
           chartType={chartType}
           instanceEvaluation={instanceEvaluation}
           showSolution={showSolution}
           textSize={textSize}
         />
-      </div>
-      <div
-        className={twMerge(
-          'order-1 flex flex-none flex-col gap-2 border-l px-4 py-2 md:order-2 md:w-64 lg:w-72 xl:w-80',
-          textSize.text
-        )}
+      </ResizablePanel>
+      <ResizableHandle withHandle />
+      <ResizablePanel
+        key={`parent-${instanceEvaluation.id}-${isCollapsed ? 'collapsed' : 'expanded'}`}
+        defaultSize={isCollapsed ? 0 : 20}
+        minSize={10}
+        collapsible
+        collapsedSize={0}
+        onCollapse={() => setIsCollapsed(true)}
```

Comment:
this should no longer be needed as it handles collapsing and size persistence automatically